.github/workflows/_deploy_kafka_test.yml

```diff
@@ -88,6 +88,31 @@ jobs:
         run: | 
           az aks get-credentials --name $CLUSTER_NAME --resource-group $RESOURCE_GROUP  --overwrite-existing
           cd main/kafka
+
+          sudo cat /root/.docker/config.json 
+          sudo mv ~/.docker/config.json /root/.docker/config.json
+          echo "docker config.json looks like the following: "
+          sudo cat /root/.docker/config.json
+          curl -LO https://github.com/microsoft/kata-containers/releases/download/3.2.0.azl1.genpolicy0/genpolicy
```

Comment:
nit: the `--fail-with-body` flag can be used with curl, so an exit code > 0 is returned if the request fails. I think other parts of the pipeline use it too

Reply:
done